### PR TITLE
docs(STRESS.md): grammatical errors

### LINKED ISSUE #2097: Fix grammatical error in stress.md
I found a grammatical error in the stress.md of the project. The error may affect the readability and clarity of the documentation.

Expected Behavior: The documentation should be free of grammatical errors and easy to understand.

Actual Behavior: The current documentation contains a grammatical error that can cause confusion for readers.

Steps to Reproduce:

1-Open stress.md
2-Navigate to line 38
3-Observe the grammatical errors in the text

If you have any additional information or context about the error, feel free to add it to the description.

### PR DESCRIPTION
**Pull request by PRIYANKjakharia**
Close #2097 

Fix grammatical errors in STRESS.md

Description:

I found a grammatical error in the STRESS.md of the project. This PR fixes the error to improve the overall readability and clarity of the documentation.

Changes:
Corrected grammatical error in line 38
Updated line 38  to reflect the correct grammar
Reasoning: The original text contained a grammatical error that could cause confusion for readers. This fix ensures that the documentation is accurate and easy to understand.

### CODE REVIEW

File: STRESS.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ kubectl -n keep exec -it keep-database-659c69689-vxhkz  -- bash -c "mysql -u roo
 
 # No Load
 ## 500k alerts - 1Gi/250m cpu: get_last_alerts 2 minutes and 30 seconds
-- Keep Backend Workers get timeout after a one minutes (500's for preset and alert endpoints)
+- Keep Backend Workers get a timeout after one minute (500's for preset and alert endpoints)
```
Comment (talboren) [inline STRESS.md:38]:
```suggestion
- Keep Backend Workers get a timeout after one minute (500 for preset and alert endpoints)
```

**Comment by talboren** [comment]
@PRIYANKjakharia please follow the instructions in the PR comments to get this PR approved.

**Comment by PRIYANKjakharia** [comment]
> @PRIYANKjakharia please follow the instructions in the PR comments to get this PR approved.

sure sir, I apologize for being naive

**Comment by PRIYANKjakharia** [comment]
I have raised an issue and mentioned it in the PR's description. anticipating for review @talboren

File: STRESS.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ kubectl -n keep exec -it keep-database-659c69689-vxhkz  -- bash -c "mysql -u roo
 
 # No Load
 ## 500k alerts - 1Gi/250m cpu: get_last_alerts 2 minutes and 30 seconds
-- Keep Backend Workers get timeout after a one minutes (500's for preset and alert endpoints)
+Keep Backend Workers get a timeout after one minute (status code 500's for preset and alert endpoints)
```
Comment (talboren) [inline STRESS.md]:
```suggestion
Keep Backend Workers get a timeout after one minute (status code 500 for preset and alert endpoints)
```

**Review by talboren — APPROVED** [review]
LGTM

**Review by Matvey-Kuk — APPROVED** [review]

**Pull request merged**